src/lib/sfge/SfgePathlessEngine.ts

```diff
@@ -1,8 +1,57 @@
+import {SfdxError, Messages} from '@salesforce/core';
 import {AbstractSfgeEngine, SfgeViolation} from "./AbstractSfgeEngine";
-import {RuleViolation} from '../../types';
-import {RuleType} from '../../Constants';
+import {Rule, RuleGroup, RuleTarget, RuleViolation, SfgeConfig} from '../../types';
+import {CUSTOM_CONFIG, MissingOptionsBehavior, RuleType} from '../../Constants';
+import {uxEvents, EVENTS} from '../ScannerEvents';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/sfdx-scanner', 'SfgeEngine');
 
 export class SfgePathlessEngine extends AbstractSfgeEngine {
+	private missingOptionsBehavior: MissingOptionsBehavior;
+
+	/**
+	 * Invokes sync/async initialization required for the engine.
+	 * @override
+	 */
+	public async init(): Promise<void> {
+		if (this.initialized) {
+			return;
+		}
+		await super.init();
+		this.missingOptionsBehavior = await this.config.getMissingOptionsBehavior(SfgePathlessEngine.ENGINE_ENUM);
```

Comment:
Since this is coming from the config, the method returns a Promise. I chose to get the value in `init()` and store it as a class property, because the alternative was to make `shouldEngineRun()` return a Promise and change it everywhere it's used.
The biggest impact of that choice is that if `missingBehaviorOption` isn't one of the valid options, running commands will immediately error out. Had I done it the other way, the error would only happen when actually attempting to run GraphEngine via `scanner:run`.